packages/synapse-react-client/src/components/SynapseChat/SynapseChatInteraction.tsx

```diff
@@ -73,11 +80,31 @@ export const SynapseChatInteraction: React.FunctionComponent<
               borderRadius: '10px',
               padding: '10px',
               maxWidth: '100%',
+              whiteSpace: 'pre-wrap',
             }}
           />
         </ListItem>
       )}
-      {!chatResponseText && <SkeletonParagraph numRows={3} />}
```

Comment:
Add additional loading UI.  I could not reproduce the Firefox issue where the skeleton paragraph was hidden, but I added a spinner and some text